pandas/tests/groupby/test_groupby.py

```diff
@@ -2552,7 +2550,6 @@ def test_groupby_numerical_stability_sum_mean(func, values):
     tm.assert_frame_equal(result, expected)
 
 
-@pytest.mark.xfail(not IS64, reason="GH#38778: fail on 32-bit system")
```

Comment:
what fixed this?  is it possible to do independently of cmake?

Reply:
Didn’t dive too deep into the 32 but stuff. I’m guessing something with the compilation flags would have done this. Here we enforce C99 standard behavior explicitly and whatever CMake chooses to send as flags to the compiler doesn’t necessarily match what distutils might have chosen